.travis.yml

```diff
@@ -8,11 +8,19 @@ services:
 before_install:
   - docker run -d -p 5000:5000 --restart=always --name registry registry:2
 
-jdk:
-  - oraclejdk8
+jdk: oraclejdk8
 
-script:
-  - ./gradlew build -s
+script: ./gradlew build -s
+
+jobs:
+  include:
+    - stage: release
+      script: skip
+      deploy:
+        provider: script
+        script: ./gradlew assemble bintrayUpload
```

Comment:
Should probably also use the option `-s` to render stack trace in case something goes wrong. That'll make it easier to debug issues. Is `assemble` still needed here? The previous script execution `./gradlew build -s` should have take care of it. Here we should also publish the Javadocs.